src/item.cpp

```diff
@@ -3373,7 +3374,34 @@ std::string item::tname( unsigned int quantity, bool with_prefix, unsigned int t
 {
     std::stringstream ret;
 
+    int dirt_level = static_cast<int>( get_var( "dirt", 0 ) / 2000 );
+    std::string dirt_symbol;
     // TODO: MATERIALS put this in json
+
+    // these symbols are a rough estimation of fouling in a gun. This appears instead of "faulty"
```

Comment:
```suggestion
    // these symbols are unicode square characeters of different heights, representing a rough
    // estimation of fouling in a gun. This appears instead of "faulty"
```